hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsScheduler.java

```diff
@@ -362,10 +362,17 @@ private ExecutorInfo createExecutor(String taskIdName, String nodeName, String e
                   String.format("http://%s:%d/%s", hdfsFrameworkConfig.getFrameworkHostAddress(),
                     confServerPort,
                     HDFSConstants.HDFS_CONFIG_FILE_NAME))
+                .build(),
```

Comment:
Let's not hard code this. This should be configurable with a default value. Please see HdfsFrameworkConfig